PelotonEppSdk/Classes/UriFactory.cs

```diff
@@ -4,11 +4,13 @@
 
 namespace PelotonEppSdk.Classes
 {
-    public class TestUriFactory
```

Comment:
In a future release this may need to change back to public. I envisioned us having a static variable which switched us between test and prod.

I.E. TestUri.Mode = Mode.Production
TestUri.Mode = Mode.Test